Dockerfile

```diff
@@ -17,79 +26,88 @@ ENV PATH /opt/conda/bin:$PATH
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && \
     echo $TZ > /etc/timezone
 
-RUN apt-get update --fix-missing && apt-get install -y wget bzip2 ca-certificates \
+RUN apt-get update && \
+    apt-get install -y wget bzip2 ca-certificates \
     libglib2.0-0 libxext6 libsm6 libxrender1 \
-    git mercurial subversion
+    git mercurial subversion \
+    tmux && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
All these clean/rm-rfs are about not keeping cache stuff around after the layer is built. It doesn't save much, but it's overall helpful.